iOS/MSMapsModulesReferenceApp/format-all.txt

```diff
@@ -0,0 +1,14 @@
+function formatCode
```

Comment:
This shouldn't be a .txt file.Is this a copy of the one in the modules directory? If so just move that script up to iOS so it will run on both.